dm/dm/master/server_test.go

```diff
@@ -2031,16 +2032,30 @@ func (t *testMaster) TestGetCfg(c *check.C) {
 	c.Assert(resp1.Result, check.IsTrue)
 	c.Assert(strings.Contains(resp1.Cfg, "name: test"), check.IsTrue)
 
-	// wrong task name
+	// not exist task name
+	notExistTaskName := "wrong"
 	req2 := &pb.GetCfgRequest{
-		Name: "haha",
+		Name: notExistTaskName,
 		Type: pb.CfgType_TaskType,
 	}
 	resp2, err := server.GetCfg(context.Background(), req2)
 	c.Assert(err, check.IsNil)
 	c.Assert(resp2.Result, check.IsFalse)
 	c.Assert(resp2.Msg, check.Equals, "task not found")
 
+	// test get task template
+	openapiTask, err := fixtures.GenNoShardOpenAPITaskForTest()
+	openapiTask.Name = notExistTaskName
+	c.Assert(err, check.IsNil)
```

Comment:
we should check the returned err before using other return values. In some case, a returned nil will panic